src/feature/hs/hs_client.c

```diff
@@ -1837,13 +1837,14 @@ hs_client_decode_descriptor(const char *desc_str,
     uint64_t current_time_period = hs_get_time_period_num(0);
     hs_build_blinded_pubkey(service_identity_pk, NULL, 0, current_time_period,
                             &blinded_pubkey);
-    hs_get_subcredential(service_identity_pk, &blinded_pubkey, subcredential);
+    hs_get_subcredential(service_identity_pk, &blinded_pubkey, &
+                         subcredential);
```

Comment:
Oh wow I thought this was a bug but also "how can this compile?" and then I noticed the `&` on the previous line.

I would recommend to fix the syntax :).

Reply:
Addressed in 975102869a.